src/app/pages/report/details/details.component.ts

```diff
@@ -97,6 +97,14 @@ export class DetailsComponent implements OnInit {
     } else {
       this.detailInputs = this.getDefaultDetailInputs();
     }
+    if (!this.detailInputs.find(input => input.type === InputType.Textarea)) {
```

Comment:
`!this.detailInputs.some(...)` ? (au moins ça renvoie toujours un booléen)

Reply:
Ok corrigé